sklearn/ensemble/bagging.py

```diff
@@ -443,8 +443,6 @@ def _get_estimators_data_draws(self, sample_mask=False, feature_mask=False):
             yield feature_draw, sample_draw
 
     @property
-    @deprecated("Attribute 'estimator_samples_' is deprecated and will be removed "
```

Comment:
I think for trees we just removed and didn't replace by anything.
If we think this is important, we could deprecate this and make it a method, to make it clear that it is generated at each access. Though I'm not sure if the deprecation is worth the sugar.